test/unit/event.js

```diff
@@ -2811,6 +2811,15 @@ QUnit.test( "preventDefault() on focusin does not throw exception", function( as
 	}, QUnit.config.testTimeout / 4 || 1000 );
 } );
 
+QUnit.test( ".on('focus', fn) on a text node doesn't throw", function( assert ) {
+	assert.expect( 1 );
+
+	jQuery( "<div></div>text<span></span>" )
```

Comment:
I'd prefer something more obvious here such as `jQuery( document.createTextNode("text") )`, but this works as long as the behavior doesn't change (do we test that, though?).

Reply:
@gibson042 Your suggestion looks good to me. What do you mean by "as long as the behavior doesn't change", though? What behavior?